authentication.md

```diff
@@ -57,8 +57,34 @@ When correctly authenticated, the response will return 200 and in this example s
 
 ## Refreshing a token
 
-The token will expire after one week. You have either to log in again or use the refresh token.  
-To be tested and documented, ticket [TC-4734](https://tradecloud.atlassian.net/browse/TC-4734)
+Access token will expire after 1 hour, refresh token after 24 hours.
+If access token expires you have either to log in again or use the refresh token.
+If refresh token expires you have to log in again.
+
+To refresh token:
```

Comment:
Suggested: "You can refresh your access token by placing an HTTP request to `/authentication/_refresh`, using the Refresh-Token header:"